.azure/templates/jobs/build/push_containers.yaml

```diff
@@ -64,3 +65,15 @@ jobs:
           DOCKER_REGISTRY: "quay.io"
           DOCKER_ORG: "strimzi"
           DOCKER_TAG: '${{ parameters.dockerTag }}'
+      - bash: "make docker_sign_manifest"
+        displayName: "Sign container manifests"
+        env:
+          BUILD_REASON: $(Build.Reason)
+          BRANCH: $(Build.SourceBranch)
+          BUILD_ID: $(Build.BuildId)
+          BUILD_COMMIT: $(Build.SourceVersion)
+          DOCKER_REGISTRY: "quay.io"
+          DOCKER_ORG: "strimzi"
+          DOCKER_TAG: '${{ parameters.dockerTag }}'
+          COSIGN_PASSWORD: $(COSIGN_PASSWORD)
```

Comment:
what's this env var? I see it's never used, or?

Reply:
It is the password to the private key. It is used by the `cosign` tool automatically, so it has to be passed but you will not see it used directly in our code.

Reply:
So I don't see it explicitly used here in your changes because that is where cosign expects the password by itself, right?

Reply:
Yes, correct. The `cosign` binary reads this directly.